readme.md

```diff
@@ -83,11 +78,11 @@ If your Homestead directory sits at the same level as your Laravel project direc
 
 Then use the following command:
 
-    $ ssh vagrant@127.0.0.1 -p 2222
+    $ vagrant ssh
 
 It would be useful to save the above command to an alias for easier access:
 
-    alias vm='ssh vagrant@127.0.0.1 -p 2222'
```

Comment:
The reason this alias is suggested is you will usually not be in the `ds-homestead` folder when working on Homestead Laravel apps, whereas the provided `vm` alias works everywhere.